admin/security.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Browser Security
+description: Learn about Coder's browser security options.
+---
+
+Coder offers two browser security features that you can choose to enable. These
+are available under **Manage** > **Admin** > **Infrastructure**.
+
+## HTTP Strict Transport Security
+
+Enabling **HTTP Strict Transport Security** sets [the
+`strict-transport-security`
+header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)
+for Coder. If you're serving Coder over TLS, you can safely enable this header
+(most ingress options add this header). After you set this header, all users
+revisiting the site will use HTTPS, since clients may no longer access the site
```

Comment:
I suggest removing the second clause ("since ...") because HSTS only has an effect for browsers that support it. HSTS is an instruction to the browser to always access the site over HTTPS, but very old browsers may not support it, and it is not generally guaranteed that clients won't access the site over HTTP, since it's a client-side feature.

Reply:
It's still a good idea to inform them though no? I mean _most_ users will be using browsers that support it.

Reply:
We can remove the `since` if it reads better, but I think _most_ of our users will be using browsers that support HSTS.

Reply:
How about: "If you are serving Coder over HTTPS, we recommend enabling [HTTP Strict Transport Security], which requests that browsers and clients to use HTTPS for all connections."

It doesn't seem like mentioning ingress controllers are useful here?

Reply:
That is fair.
The ingress thing was mentioned as it's covered by nginx by default. So anyone using that doesn't need this.

Idk, I'm ok with dropping the ingress mention, but this problem is usually resolved there.

Reply:
My reasoning for dropping the ingress mention was that we're dropping the ingress controller requirement in a future release. It doesn't hurt to enable this if a proxy in front is also adding it, right?

Reply:
It does not hurt, no